core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -150,6 +158,9 @@ private[spark] class ExecutorAllocationManager(
   // Metric source for ExecutorAllocationManager to expose internal status to MetricsSystem.
   val executorAllocationManagerSource = new ExecutorAllocationManagerSource
```

Comment:
Can it be? Isn't this something that the metrics system polls (does not look like it is used locally here)